domain/application/state/charm_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 Canonical Ltd.
+// Copyright 2024 Canonical Ltd
```

Comment:
Why? Other copyright notices include the full stop

Reply:
Sorry, bad vim command.